src/Output/Tags.hs

```diff
@@ -103,19 +103,20 @@ showTag (EqCategory k v) = (k,v)
 ---------------------------------------------------------------------
 -- TAG SEMANTICS
 
--- | Given a tag, find the ranges of identifiers it covers
-resolveTag :: StoreRead -> Tag -> (Maybe Tag, [(TargetId,TargetId)])
+-- | Given a tag, find the ranges of identifiers it covers(if it restricts the range)
+-- An empty range means an empty result, while a Nothing means a search on the entire range
+resolveTag :: StoreRead -> Tag -> (Tag, Maybe [(TargetId,TargetId)])
```

Comment:
It doesn't for packages, it searches for package prefixes, then returns the closest one it finds.